### PR TITLE
Created article for v9 and updated the methods

### PR DESCRIPTION
**Pull request by eshanrnh**

### CODE REVIEW

File: Fundamentals/Design/Rendering-Media/index-v9.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+```html
+<img src="@mediaItem.Url" />
+```
+
+### More information
+
+- [Media Picker](../../Backoffice/Property-Editors/Built-in-Property-Editors/Media-Picker/index.md)
+- [Image Cropper](../../Backoffice/Property-Editors/Built-in-Property-Editors/Image-Cropper.md)
+- [Creating a Media Type](../../Data/Creating-Media/index.md#creating-a-media-type)
```
Comment (jonat123) [inline Fundamentals/Design/Rendering-Media/index-v9.md:116]: Until the release these links should link to the V9 articles if they exist :)

**Comment by jonat123** [comment]
@eshanrnh  looks good, i added a small note in regards to  the links in the bottom :-)

File: Fundamentals/Design/Rendering-Media/index-v9.md (hunk 1 above)
Comment (eshanrnh) [reply Fundamentals/Design/Rendering-Media/index-v9.md]: @jonat123 Only the Media Picker article contains the v9 version - its updated now :)

**Comment by jonat123** [comment]
Perfect! I am merging in :D

**Pull request merged**